Dfe.Academies.Academisation.Domain/ApplicationAggregate/Application.cs

```diff
@@ -144,14 +144,14 @@ public CommandResult Submit(DateTime submissionDate)
 		return new CommandSuccessResult();
 	}
 
-	internal static CreateResult<IApplication> Create(ApplicationType applicationType,
+	internal static CreateResult Create(ApplicationType applicationType,
```

Comment:
Small github warning above.

Reply:
That's 'legacy' code (i.e. not part of my changes) so I'll leave that alone for now - maybe a tech dept item? Though there are 350+ things that need attention across the codebase.